frontend/packages/operator-lifecycle-manager/src/components/install-plan.tsx

```diff
@@ -279,12 +280,12 @@ export const InstallPlanDetails: React.SFC<InstallPlanDetailsProps> = ({ obj })
                   </dd>
                 ))}
                 <dt>Catalog Sources</dt>
-                {(_.get(obj.status, 'catalogSources') || []).map((catalogName) => (
+                {_.get(obj, 'status.catalogSources', []).map((catalogName) => (
```

Comment:
I'd leave this

```suggestion
                {(_.get(obj, 'status.catalogSources') || []).map((catalogName) => (
```

Just in case we get `catalogSources: null`